### DISCUSSION TITLE
Add customer and time on list of Alltickets

### DISCUSSION THREAD

**Discussion by ddacosta1** [Peppermint-Lab/peppermint#378] [Ideas]
I think that to better sort and manage customer tickets, we should have the customer in the ticket list.
And add the time spent on the ticket to better calculate the time spent per customer.
Like in the image:
![Exemple](https://github.com/user-attachments/assets/0e974736-984d-42c0-8f18-5ade859a82cf)

**Comment by potts99** [comment]
done 👍

**Comment by ddacosta1** [comment]
I updated Pepermint and now i have this error in logs:


2024-10-22T11:58:02: PM2 log: App [api:1] exited with code [1] via signal [SIGINT]

2024-10-22T11:58:05: PM2 log: App [api:1] starting in -fork mode-

2024-10-22T11:58:05: PM2 log: App [api:1] online

Error: Command failed: npx prisma migrate deploy


Error: P3009

migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

The `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed

    at genericNodeError (node:internal/errors:984:15)

    at wrappedFn (node:internal/errors:538:14)

    at ChildProcess.exithandler (node:child_process:422:12)

    at ChildProcess.emit (node:events:519:28)

    at maybeClose (node:internal/child_process:1105:16)

    at ChildProcess._handle.onexit (node:internal/child_process:305:5) {

  code: 1,

  killed: false,

  signal: null,

  cmd: 'npx prisma migrate deploy'

}

Environment variables loaded from .env

Prisma schema loaded from src/prisma/schema.prisma

Datasource "db": PostgreSQL database "peppermint", schema "public" at "peppermint_postgres"

48 migrations found in prisma/migrations


Error: P3009

migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

The `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed

2024/10/22 01:58PM 50 pid=1801 hostname=14f79ae5d7f8 err={"type":"Error","message":"Command failed: npx prisma migrate deploy\nError: P3009\n\nmigrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve\nThe `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed\n\n\n","stack":"Error: Command failed: npx prisma migrate deploy\nError: P3009\n\nmigrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve\nThe `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed\n\n\n\n    at genericNodeError (node:internal/errors:984:15)\n    at wrappedFn (node:internal/errors:538:14)\n    at ChildProcess.exithandler (node:child_process:422:12)\n    at ChildProcess.emit (node:events:519:28)\n    at maybeClose (node:internal/child_process:1105:16)\n    at ChildProcess._handle.onexit (node:internal/child_process:305:5)","code":1,"killed":false,"signal":null,"cmd":"npx prisma migrate deploy"} msg=Command failed: npx prisma migrate deploy
Error: P3009

migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve
The `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed

**Reply by potts99** [reply]
investigating now thanks

**Reply by potts99** [reply]
should be fixed @ddacosta1

**Reply by ddacosta1** [reply]
Nope

`2024-10-22T14:23:57: PM2 log: App [api:1] exited with code [1] via signal [SIGINT]

2024-10-22T14:24:00: PM2 log: App [api:1] starting in -fork mode-

2024-10-22T14:24:00: PM2 log: App [api:1] online

Error: Command failed: npx prisma migrate deploy


Error: P3009

migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

The `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed

    at genericNodeError (node:internal/errors:984:15)

    at wrappedFn (node:internal/errors:538:14)

    at ChildProcess.exithandler (node:child_process:422:12)

    at ChildProcess.emit (node:events:519:28)

    at maybeClose (node:internal/child_process:1105:16)

    at ChildProcess._handle.onexit (node:internal/child_process:305:5) {

  code: 1,

  killed: false,

  signal: null,

  cmd: 'npx prisma migrate deploy'

}


Error: P3009

migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

The `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed

Environment variables loaded from .env

Prisma schema loaded from src/prisma/schema.prisma

Datasource "db": PostgreSQL database "peppermint", schema "public" at "peppermint_postgres"

48 migrations found in prisma/migrations

2024/10/22 04:24PM 50 pid=512 hostname=7b2eb5f29f93 err={"type":"Error","message":"Command failed: npx prisma migrate deploy\nError: P3009\n\nmigrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve\nThe `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed\n\n\n","stack":"Error: Command failed: npx prisma migrate deploy\nError: P3009\n\nmigrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve\nThe `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed\n\n\n\n    at genericNodeError (node:internal/errors:984:15)\n    at wrappedFn (node:internal/errors:538:14)\n    at ChildProcess.exithandler (node:child_process:422:12)\n    at ChildProcess.emit (node:events:519:28)\n    at maybeClose (node:internal/child_process:1105:16)\n    at ChildProcess._handle.onexit (node:internal/child_process:305:5)","code":1,"killed":false,"signal":null,"cmd":"npx prisma migrate deploy"} msg=Command failed: npx prisma migrate deploy
Error: P3009

migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve
The `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed`

**Reply by potts99** [reply]
have you pulled the latest image?

**Reply by ddacosta1** [reply]
Yes, i will try

**Reply by ddacosta1** [reply]
Yes Latest: 
![{17B8331B-9D79-4F6A-A95A-7845DF11EA8F}](https://github.com/user-attachments/assets/a7e11e03-5209-4816-b596-726c2f8293a0)

**Reply by ddacosta1** [reply]
2024-10-22T15:20:18: PM2 log: App [client:0] starting in -fork mode-

2024-10-22T15:20:18: PM2 log: App [api:1] starting in -fork mode-

2024-10-22T15:20:18: PM2 log: App [client:0] online

2024-10-22T15:20:18: PM2 log: App [api:1] online

  ▲ Next.js 13.5.7

  - Local:        http://2cc250b74caf:3000

  - Network:      http://172.23.0.2:3000

 ✓ Ready in 1884ms

Error: Command failed: npx prisma migrate deploy


Error: P3009

migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

The `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed

    at genericNodeError (node:internal/errors:984:15)

    at wrappedFn (node:internal/errors:538:14)

    at ChildProcess.exithandler (node:child_process:422:12)

    at ChildProcess.emit (node:events:519:28)

    at maybeClose (node:internal/child_process:1105:16)

    at ChildProcess._handle.onexit (node:internal/child_process:305:5) {

  code: 1,

  killed: false,

  signal: null,

  cmd: 'npx prisma migrate deploy'

}

Environment variables loaded from .env

Prisma schema loaded from src/prisma/schema.prisma

Datasource "db": PostgreSQL database "peppermint", schema "public" at "peppermint_postgres"

48 migrations found in prisma/migrations


Error: P3009

migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

The `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed

2024/10/22 05:20PM 50 pid=20 hostname=2cc250b74caf err={"type":"Error","message":"Command failed: npx prisma migrate deploy\nError: P3009\n\nmigrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve\nThe `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed\n\n\n","stack":"Error: Command failed: npx prisma migrate deploy\nError: P3009\n\nmigrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve\nThe `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed\n\n\n\n    at genericNodeError (node:internal/errors:984:15)\n    at wrappedFn (node:internal/errors:538:14)\n    at ChildProcess.exithandler (node:child_process:422:12)\n    at ChildProcess.emit (node:events:519:28)\n    at maybeClose (node:internal/child_process:1105:16)\n    at ChildProcess._handle.onexit (node:internal/child_process:305:5)","code":1,"killed":false,"signal":null,"cmd":"npx prisma migrate deploy"} msg=Command failed: npx prisma migrate deploy
Error: P3009

migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve
The `20241021230158_redirecturi` migration started at 2024-10-22 11:53:04.615321 UTC failed

**Reply by ddacosta1** [reply]
To solve the problem i had to recreate my DB (I have a backup for tests)
In Closed Tickets i don't see the modification..
![image](https://github.com/user-attachments/assets/0fd842e0-9408-4759-9cf7-0633683c6f3a)

And on the Dashboard maybe a thing like that ?

![image](https://github.com/user-attachments/assets/56331898-472c-434b-a268-c9fa333738b7)
